autoload/cmake4vim.vim

```diff
@@ -32,10 +32,27 @@ function! cmake4vim#CompleteTarget(arg_lead, cmd_line, cursor_pos) abort
     return join(l:sorted_targets, "\n")
 endfunction
 
+function! cmake4vim#populateDefaultCMakeVariants() abort
+    for build_type in filter( utils#cmake#getDefaultBuildTypes(), "v:val !=# ''" )
+        if ( !has_key( g:cmake_variants, build_type ) )
+            let g:cmake_variants[ build_type ] =
+                \ {
+                \   'cmake_build_type' : build_type,
```

Comment:
What if user in own config has changed build type for default target. I think we should skip these changes.